docs/conf/kenya/2025/tickets.rst

```diff
@@ -0,0 +1,125 @@
+:template: {{year}}/generic.html
+:og:image: _static/conf/images/headers/{{shortcode}}-{{year}}-opengraph.jpg
+:banner: _static/conf/images/headers/2025/tickets.jpg
+
+Tickets
+=======
+
+{% if flagticketsonsale %}
+
+**Tickets are on sale now!**
+
+We're excited to invite you to our {{ year }} conference in {{ city }}.
+
+{% elif flagsoldout %}
+
+**Tickets are sold out!**
+
+{% else %}
+
+**Tickets will be available in {{ date.tickets_live }}.**
+
+{% endif %}
+
+Ticket Information
+------------------
+
+Write the Docs {{ name }} {{ year }} is an in-person conference. Each ticket includes:
+
+* Entry to all conference events and activities
+* Snacks and drinks on event days (Sunday-Tuesday)
+* Welcome Reception and Social Event with light snacks and drinks
+* Wifi throughout the event
+* Meeting lots of fantastic people in a spacious, inviting venue
+
+All attendees are required to abide by our `Code of Conduct <https://www.writethedocs.org/code-of-conduct/>`_.
+
+Refund Policy
+-------------
+
+Refunds will be offered with a 10% processing fee until 2 weeks before the conference.
```

Comment:
This is a bit strange with the tickets being free.

Reply:
Yea, just forgot to remove it :)